### ISSUE TITLE
KeY may freeze when interrupting

### ISSUE THREAD

**Issue by wadoon** [KeYProject/key#727]
This issue was created at [git.key-project.org](https://git.key-project.org/key/key/-/issues/727) where the discussions are preserved.
    
----
    
* Mantis: [MT-1598](http://i12www.ira.uka.de/~klebanov/mantis/view.php?id=1598)
* Submitted on: 2017-03-23 by (at)mulbrich
* Updated: 2017-03-23

### Description

> I ran an interactive automatic KeY proof and stopped and reengaged the automation several time.
> 
> After a few switches, KeY went into a deadlock where no user input was possible any longer.
> 
> Apparently the busy waiting in MediatorProofControl.waitWhileAutoMode has not been implemented thread-safe.
> (Why is busy waiting used anyway?)
> 
> See attached thread stack for more information.

### Steps to reproduce

```
n/a
```

## Files

  
`jstack.txt`:

```
2017-03-23 15:38:22
Full thread dump Java HotSpot(TM) Server VM (25.121-b13 mixed mode):

"Attach Listener" #86 daemon prio=9 os_prio=0 tid=0xa6174c00 nid=0x1e2c waiting on condition [0x00000000]
   java.lang.Thread.State: RUNNABLE

"JDWP Command Reader" #85 daemon prio=10 os_prio=0 tid=0x09f3fc00 nid=0x1801 runnable [0x00000000]
   java.lang.Thread.State: RUNNABLE

"SwingWorker-pool-1-thread-10" #77 daemon prio=5 os_prio=0 tid=0x22b04c00 nid=0x17c3 waiting on condition [0x1a1fe000]
   java.lang.Thread.State: WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x28edb710> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

"SwingWorker-pool-1-thread-9" #75 daemon prio=5 os_prio=0 tid=0x22b14c00 nid=0x17c1 waiting on condition [0x1e0fe000]
   java.lang.Thread.State: WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x28edb710> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

"SwingWorker-pool-1-thread-8" #74 daemon prio=5 os_prio=0 tid=0x1ffd8400 nid=0x17c0 waiting on condition [0x1ddba000]
   java.lang.Thread.State: WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x28edb710> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

"SwingWorker-pool-1-thread-7" #72 daemon prio=5 os_prio=0 tid=0x22b12c00 nid=0x17be waiting on condition [0x1e2ad000]
   java.lang.Thread.State: WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x28edb710> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

"SwingWorker-pool-1-thread-6" #62 daemon prio=5 os_prio=0 tid=0x22b01400 nid=0x1786 waiting on condition [0x1eaa1000]
   java.lang.Thread.State: WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x28edb710> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

"SwingWorker-pool-1-thread-5" #60 daemon prio=5 os_prio=0 tid=0x22b02000 nid=0x1780 waiting on condition [0x2266e000]
   java.lang.Thread.State: WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x28edb710> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

"SwingWorker-pool-1-thread-4" #53 daemon prio=5 os_prio=0 tid=0x22ee5c00 nid=0x1733 waiting on condition [0x2202f000]
   java.lang.Thread.State: WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x28edb710> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

"SwingWorker-pool-1-thread-3" #49 daemon prio=5 os_prio=0 tid=0x1ffd7000 nid=0x172f waiting on condition [0x23214000]
   java.lang.Thread.State: WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x28edb710> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

"JDWP Transport Listener: dt_socket" #45 daemon prio=10 os_prio=0 tid=0xa638ac00 nid=0x1725 runnable [0x00000000]
   java.lang.Thread.State: RUNNABLE

"SwingWorker-pool-1-thread-2" #44 daemon prio=5 os_prio=0 tid=0x22eca400 nid=0x170f waiting on condition [0x1e2fe000]
   java.lang.Thread.State: WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x28edb710> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

"DestroyJavaVM" #24 prio=5 os_prio=0 tid=0xb6008c00 nid=0x14d2 waiting on condition [0x00000000]
   java.lang.Thread.State: RUNNABLE

"SwingWorker-pool-1-thread-1" #23 daemon prio=5 os_prio=0 tid=0x1ffa9c00 nid=0x14f4 waiting on condition [0x1e02d000]
   java.lang.Thread.State: WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x28edb710> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

"TimerQueue" #22 daemon prio=5 os_prio=0 tid=0x1ff32400 nid=0x14f3 waiting on condition [0x1e07e000]
   java.lang.Thread.State: WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x28e61cf0> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.DelayQueue.take(DelayQueue.java:211)
	at javax.swing.TimerQueue.run(TimerQueue.java:174)
	at java.lang.Thread.run(Thread.java:745)

"AWT-EventQueue-0" #20 prio=6 os_prio=0 tid=0x2258d800 nid=0x14f0 waiting on condition [0x21fdd000]
   java.lang.Thread.State: TIMED_WAITING (sleeping)
	at java.lang.Thread.sleep(Native Method)
	at de.uka.ilkd.key.ui.MediatorProofControl.waitWhileAutoMode(MediatorProofControl.java:99)
	at de.uka.ilkd.key.control.AbstractProofControl.stopAndWaitAutoMode(AbstractProofControl.java:576)
	at de.uka.ilkd.key.gui.actions.AutoModeAction.actionPerformed(AutoModeAction.java:182)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348)
	at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
	at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
	at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:252)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:290)
	at java.awt.Component.processMouseEvent(Component.java:6533)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6298)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4889)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at de.uka.ilkd.key.gui.MainWindow$GlassPaneListener.dispatchForCurrentComponent(MainWindow.java:1267)
	at de.uka.ilkd.key.gui.MainWindow$GlassPaneListener.redispatchMouseEvent(MainWindow.java:1225)
	at de.uka.ilkd.key.gui.MainWindow$GlassPaneListener.mouseReleased(MainWindow.java:1219)
	at java.awt.Component.processMouseEvent(Component.java:6533)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6298)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4889)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4525)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
	at java.awt.Container.dispatchEventImpl(Container.java:2280)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

"AWT-Shutdown" #21 prio=5 os_prio=0 tid=0x2258c400 nid=0x14ef in Object.wait() [0x220ad000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.lang.Object.wait(Object.java:502)
	at sun.awt.AWTAutoShutdown.run(AWTAutoShutdown.java:295)
	- locked <0x28e620d0> (a java.lang.Object)
	at java.lang.Thread.run(Thread.java:745)

"Timer-0" #18 daemon prio=5 os_prio=0 tid=0x22525400 nid=0x14ee in Object.wait() [0x220fe000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.util.TimerThread.mainLoop(Timer.java:552)
	- locked <0x28e62258> (a java.util.TaskQueue)
	at java.util.TimerThread.run(Timer.java:505)

"AWT-XAWT" #15 daemon prio=6 os_prio=0 tid=0x22be4c00 nid=0x14eb waiting on condition [0x226bf000]
   java.lang.Thread.State: TIMED_WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x297c2a18> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
	at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2163)
	at sun.awt.SunToolkit.awtLockWait(SunToolkit.java:273)
	at sun.awt.X11.XlibWrapper.XNextSecondaryLoopEvent(Native Method)
	at sun.awt.X11.XToolkit.run(XToolkit.java:557)
	at sun.awt.X11.XToolkitThreadBlockedHandler.enter(XToolkitThreadBlockedHandler.java:49)
	at sun.awt.datatransfer.DataTransferer.convertData(DataTransferer.java:2344)
	at sun.awt.X11.XSelection.convertAndStore(XSelection.java:487)
	at sun.awt.X11.XSelection.handleSelectionRequest(XSelection.java:564)
	at sun.awt.X11.XSelection.access$500(XSelection.java:45)
	at sun.awt.X11.XSelection$SelectionEventHandler.dispatchEvent(XSelection.java:746)
	at sun.awt.X11.XToolkit.dispatchEvent(XToolkit.java:516)
	at sun.awt.X11.XToolkit.run(XToolkit.java:616)
	at sun.awt.X11.XToolkit.run(XToolkit.java:532)
	at java.lang.Thread.run(Thread.java:745)

"Java2D Disposer" #13 daemon prio=10 os_prio=0 tid=0x22bcf800 nid=0x14ea in Object.wait() [0x2271b000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:143)
	- locked <0x28e62780> (a java.lang.ref.ReferenceQueue$Lock)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:164)
	at sun.java2d.Disposer.run(Disposer.java:148)
	at java.lang.Thread.run(Thread.java:745)

"Service Thread" #11 daemon prio=9 os_prio=0 tid=0x23bc5000 nid=0x14e8 runnable [0x00000000]
   java.lang.Thread.State: RUNNABLE

"C1 CompilerThread2" #10 daemon prio=9 os_prio=0 tid=0x23bb9c00 nid=0x14e7 waiting on condition [0x00000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread1" #9 daemon prio=9 os_prio=0 tid=0x23bb8400 nid=0x14e6 waiting on condition [0x00000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread0" #8 daemon prio=9 os_prio=0 tid=0x23bb6c00 nid=0x14e5 waiting on condition [0x00000000]
   java.lang.Thread.State: RUNNABLE

"JDWP Event Helper Thread" #7 daemon prio=10 os_prio=0 tid=0x23bb5000 nid=0x14e4 runnable [0x00000000]
   java.lang.Thread.State: RUNNABLE

"Signal Dispatcher" #5 daemon prio=9 os_prio=0 tid=0x23ba9000 nid=0x14e2 runnable [0x00000000]
   java.lang.Thread.State: RUNNABLE

"Surrogate Locker Thread (Concurrent GC)" #4 daemon prio=9 os_prio=0 tid=0x23ba7c00 nid=0x14e1 waiting on condition [0x00000000]
   java.lang.Thread.State: RUNNABLE

"Finalizer" #3 daemon prio=8 os_prio=0 tid=0x23b8dc00 nid=0x14e0 in Object.wait() [0x23307000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:143)
	- locked <0x28e638e8> (a java.lang.ref.ReferenceQueue$Lock)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:164)
	at java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:209)

"Reference Handler" #2 daemon prio=10 os_prio=0 tid=0x23b8b000 nid=0x14df in Object.wait() [0x23358000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.lang.Object.wait(Object.java:502)
	at java.lang.ref.Reference.tryHandlePending(Reference.java:191)
	- locked <0x28e63a88> (a java.lang.ref.Reference$Lock)
	at java.lang.ref.Reference$ReferenceHandler.run(Reference.java:153)

"VM Thread" os_prio=0 tid=0x23b86800 nid=0x14de runnable 

"Gang worker#0 (Parallel GC Threads)" os_prio=0 tid=0xb6013800 nid=0x14d3 runnable 

"Gang worker#1 (Parallel GC Threads)" os_prio=0 tid=0xb6014c00 nid=0x14d4 runnable 

"Gang worker#2 (Parallel GC Threads)" os_prio=0 tid=0xb6016000 nid=0x14d5 runnable 

"Gang worker#3 (Parallel GC Threads)" os_prio=0 tid=0xb6017400 nid=0x14d6 runnable 

"G1 Main Concurrent Mark GC Thread" os_prio=0 tid=0xb6077c00 nid=0x14dc runnable 

"Gang worker#0 (G1 Parallel Marking Threads)" os_prio=0 tid=0xb6079000 nid=0x14dd runnable 

"G1 Concurrent Refinement Thread#0" os_prio=0 tid=0xb605e400 nid=0x14db runnable 

"G1 Concurrent Refinement Thread#1" os_prio=0 tid=0xb605d000 nid=0x14da runnable 

"G1 Concurrent Refinement Thread#2" os_prio=0 tid=0xb605bc00 nid=0x14d9 runnable 

"G1 Concurrent Refinement Thread#3" os_prio=0 tid=0xb605a800 nid=0x14d8 runnable 

"G1 Concurrent Refinement Thread#4" os_prio=0 tid=0xb6059000 nid=0x14d7 runnable 

"VM Periodic Task Thread" os_prio=0 tid=0x23bc7800 nid=0x14e9 waiting on condition 

JNI global references: 7722


```

## Associated Bugs

* Relates to MT-1580 with 

## History

* (at)mulbrich -- (`NEW_BUG`) _2017-03-23_

* (at)mulbrich -- (`FILE_ADDED`) _2017-03-23_

* (at)mulbrich -- (`BUG_ADD_RELATIONSHIP`) _2017-03-23_

## Attributes

* Category: GUI
* Status: NEW
* Severity: MINOR
* OS: 
* Target Version: 
* Resolution: OPEN
* Priority: NORMAL
* Reproducibility: RANDOM
* Platform: 
* Commit: 90dde5a1f3b2857ec76a728ce080f1a88f0e4aa7
* Build: 
* Tags []
* Labels:  ~GUI  ~Bug  ~NORMAL  

---

[View in Mantis](http://i12www.ira.uka.de/~klebanov/mantis/view.php?id=1598)
    
----
    
Information: 
* created_at: 2017-05-29T02:45:03.501Z
* updated_at: 2017-08-15T14:07:05.529Z
* closed_at: None  (closed_by: )
* milestone: 
* user_notes_count: 1

**Comment by FliegendeWurst** [comment]
Duplicate of #407
